### DISCUSSION TITLE
Importing and using country flag icons in vue app

### DISCUSSION THREAD

**Discussion by Ash-the-practical-programmer** [primefaces/community#2446] [PrimeVue] [answered]
There had been no illustration given in primevue's official documentation regarding country flag icons , how to import exactly n how to use them ??

**Comment by j0rgedev** [comment, marked as answer]
Hello @Ash-the-practical-programmer. If you mean country flag icons like those in the [Select Component Template](https://primevue.org/select/#template), you'll need to have the flag.css file that PrimeVue uses. Check this stackblitz to see its implementation: [Example](https://stackblitz.com/run?file=src%2Fflags.css)

**Comment by platform-kit** [comment]
@j0rgedev the link you provided no longer works. Is there no official documentation of this flag feature?

**Reply by cagataycivici** [reply]
Flag is not a feature, it is a demo. You can use any flag library with the option template.

**Reply by platform-kit** [reply]
@cagataycivici I see. That's unclear from the Select documentation page, might want to tweak the verbiage.
